tests/integration/surcharges.test.js

```diff
@@ -123,23 +122,24 @@ describe('POST /surcharges', () => {
   describe('Other roles', () => {
```

Comment:
peut tu mettre un describe autour de tous les it au dessus pour isoler le cas client_admin ? et aussi ne pas lancer le beforeeach avant les test de other roles

Reply:
et idem pour les autres routes

Reply:
done